src/Room.java

```diff
@@ -0,0 +1,71 @@
+public class Room {
+
+    private double squareMeter;
+    private double temperature;
+    private boolean airConditioning;
+    private double limitTemperature;
+
+    public Room(double temperature, boolean airConditioning, double limitTemperature) {
+        this.squareMeter = squareMeter;
+        this.temperature = temperature;
+        this.airConditioning = airConditioning;
+        this.limitTemperature = limitTemperature;
+    }
+
+    public double getSquareMeter() {
+        return squareMeter;
+    }
+
+    public void setSquareMeter(double squareMeter) {
+        this.squareMeter = squareMeter;
+    }
+
+    public double getTemperature() {
+        return temperature;
+    }
+
+    public void setTemperature(double temperature) {
+        this.temperature = temperature;
+    }
+
+    public boolean isAirConditioning() {
+        return airConditioning;
+    }
+
+    public void setAirConditioning(boolean airConditioning) {
+        this.airConditioning = airConditioning;
+    }
+
+    public double getLimitTemperature() {
+        return limitTemperature;
+    }
+
+    public void setLimitTemperature(double limitTemperature) {
+        this.limitTemperature = limitTemperature;
+    }
+
+    boolean decreaseTemperature() {
+        if ((getTemperature() >= getLimitTemperature()) && airConditioning) {
+            return true;
+        }
+        return false;
+    }
+
+    boolean hasAirConditioning() {
+        if (airConditioning) {
+            return decreaseTemperature();
+        }
+        return false;
+    }
+
+    public double coolingRoom(double getSquareMeter) {
```

Comment:
jesli chcesz obnizać temperature pokoju w zaleznosci od metrazu to ta metoda powinna nie miec zadnego argumentu tylko czytać metraz z pola `squareMeter`. Dodatkowo powinienies wtedy uzyc tej metody w metodzie `decreaseTemperature`